vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -414,28 +742,71 @@ EXPORT(int, sceGxmCreateContext, const SceGxmContextParams *params, Ptr<SceGxmCo
     if (!params || !context)
         return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
 
-    *context = alloc<SceGxmContext>(host.mem, __FUNCTION__);
-    if (!*context) {
-        return RET_ERROR(SCE_GXM_ERROR_OUT_OF_MEMORY);
+    if (params->hostMemSize < sizeof(SceGxmContext)) {
```

Comment:
Maybe we should check uniqueness of immeidate context now. Cause, we removed gxm.is_in_scene.

Reply:
Why here?

Reply:
Two contexts might be created, if weird things happen. (e.g. race) It's for debugging purpose. If you don't like the idea, just ignore it.

Reply:
I'm not sureee, I don't understand why check the uniqueness, since all have same size.

Reply:
I meant checking one immediate context for one processor. Sorry for confusing line reference.

Reply:
This I will do later. Seems to need add some more variables.